db/patterns/bing_maps.eno

```diff
@@ -1,10 +1,22 @@
-name: Bing Maps
+name: Bing/Azure Maps
 category: customer_interaction
-website_url: 
+website_url: https://www.bingmapsportal.com
 organization: microsoft
 
 --- domains
 virtualearth.net
+r.bing.com
+atlas.microsoft.com
 --- domains
 
+--- filters
+r.bing.com/rp^$script
+www.bing.com/api/maps/
+www.bing.com/maps/sdk
+www.bing.com/maps/geotfe
+www.bing.com/maps/instrumentation
+www.bing.com/fd/ls/lsp.aspx
+--- filters
+
+
 ghostery_id: 3400
```

Comment:
Based on my understanding, your original rules should be matching correctly, but the TrackerDB command line tool has some problem (unless I had `$1p` or `$3p`). The change here is only to prepend "||", which is more common in TrackerDB. I think, it should be affect the matching in a negative way:

```suggestion
--- filters
||r.bing.com/rp^$script
||www.bing.com/api/maps/
||www.bing.com/maps/sdk
||www.bing.com/maps/geotfe
||www.bing.com/maps/instrumentation
||www.bing.com/fd/ls/lsp.aspx
--- filters

ghostery_id: 3400
```